skimage/morphology/watershed.py

```diff
@@ -55,6 +174,8 @@ def watershed(image, markers, connectivity=None, offset=None, mask=None):
     mask: ndarray of bools or 0s and 1s, optional
         Array of same shape as `image`. Only points at which mask == True
         will be labeled.
+    compactness : float, optional
+        Use compact watershed [3]_ with given compactness parameter.
```

Comment:
Please, add intuitive description of what this value does, e.g. "A lower value results in more compact regions, etc."